src/mongocxx/test/spec/unified_tests/operations.cpp

```diff
@@ -1296,11 +1296,9 @@ document::value operations::run(entity::map& entity_map,
         return empty_doc;
     }
     if (name == "assertSessionNotDirty") {
-        // TODO CXX-2227: bump minimum required libmongoc version to 1.18
```

Comment:
Should I implement these unified test runner functions now? They are needed to run the `poc-sessions` test in unified format.

Reply:
I'll just do them now, actually. Doesn't make much sense to wait.